src/main/scala/com/microsoft/ml/spark/featurize/AssembleFeatures.scala

```diff
@@ -443,7 +443,7 @@ class AssembleFeaturesModel(val uid: String,
         val outputColsKeys = columnNamesToFeaturize.categoricalColumns.values
         val inputCols = inputColsKeys.toArray
         val outputCols = outputColsKeys.toArray
-        val ohe = new OneHotEncoderEstimator()
+        val ohe = new OneHotEncoder()
```

Comment:
is there some way to make this be based on the currently used spark version?

Reply:
Can you do something like...
val ohe = if (SparkContext.version < 3.0) new OneHotEncoderEstimator() else new OneHotEncoder()

Reply:
Yeah @dciborow sounds good, do that!